app/views/records/_active_filters.html.haml

```diff
@@ -1,19 +1,20 @@
-/
-  \# The majority of The Supplejack Website code is Crown copyright (C) 2014, New Zealand Government,
-  \# and is licensed under the GNU General Public License, version 3. Some components are
-  \# third party components that are licensed under the MIT license or other terms.
-  \# See https://github.com/DigitalNZ/supplejack_website for details.
-  \#
-  \# Supplejack was created by DigitalNZ at the National Library of NZ and the Department of Internal Affairs.
-  \# http://digitalnz.org/supplejack
+/ # The majority of The Supplejack Website code is Crown copyright (C) 2014,
+/ # New Zealand Government, and is licensed under the GNU General Public License,
+/ #version 3. Some components are third party components that are licensed under
+/ # the MIT license or other terms.
+/ # See https://github.com/DigitalNZ/supplejack_website for details.
+/ #
+/ # Supplejack was created by DigitalNZ at the National Library of NZ and
+/ # the Department of Internal Affairs.
+/ # http://digitalnz.org/supplejack
 - params['i'].each do |facet, value|
-  = content_tag :button, value,          |
+  = content_tag :button, date_parser_for(value),          |
     class: "filter-unit",                |
     id: "target-#{filter_class(value)}", |
     data: {facet: facet, filter: value}  |
 - params['or'].each do |facet, values|
   - values.each do |value|
-    = content_tag :button, value,          |
+    = content_tag :button, date_parser_for(value),          |
```

Comment:
Don't use the `|` character to split up lines. Wrap on commas or extract code into helper.